src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1 +1 @@
-
+Microsoft.CodeAnalysis.Editing.SyntaxGenerator.WithTypeConstraint(Microsoft.CodeAnalysis.SyntaxNode declaration, string typeParameterName, Microsoft.CodeAnalysis.Editing.SpecialTypeConstraintKind kinds, System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.SyntaxNode> types = null) -> Microsoft.CodeAnalysis.SyntaxNode
```

Comment:
no clue what that is.  removing.

Reply:
oh wait.  nevermind.  So this is a case of "does not need api review".  Specifically, an abstract member on a non-publicly-inheritable type became non-abstract.  There is no binary nor source break here.  All existing consumers that potentially call into this will continue to do so without issue.

Reply:
this should now be clearer seeing the 'removed' and 'added' item.